guardian/support-frontend
# Remove DA postcodes that we don't serve for home delivery

## What are you doing in this PR?
Removes some DA-area postcodes that we don't serve for Home Delivery:
<img width="1243" alt="Screenshot 2024-06-10 at 08 15 21" src="https://github.com/guardian/support-frontend/assets/23298731/f814f17a-5191-4bd5-80b0-bae74b998826">

This is from the most up to date list we currently have (although we are trying to get an even more up to date one to review all postcode areas).

## Why are you doing this?
We don't get a lot of business from the Gravesend area, but a customer recently purchased a print delivery subscription and was wrongly assigned to Home Delivery rather than National Delivery.

Via National Delivery, we do serve these postcodes.

## How to test
Try to checkout with a subscription in the DA12 (or another removed postcode) and verify that the National Delivery product is being assigned, not Home Delivery (if the Delivery Provider section is displayed, it's National Delivery).

<img width="1151" alt="Screenshot 2024-06-10 at 09 04 44" src="https://github.com/guardian/support-frontend/assets/23298731/076579ba-4e4f-4934-acfd-cbfe55169487">
